### ISSUE TITLE
[Bug]: lxappearance does not run

### ISSUE THREAD

**Issue by AspirusFromScratch** [termux/x11-packages#424]
### Problem description

lxappearance does not run with no errors

### What steps will reproduce the bug?

run the lxappearance program 

### What is the expected behavior?

lxappearance should run

### System information

termux-info:

```
Application version:
0.113
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://termux.org/packages/ stable main
# game-repo (sources.list.d/game.list)
deb https://packages.termux.org/apt/termux-games games stable
# science-repo (sources.list.d/science.list)
deb https://packages.termux.org/apt/termux-science science stable
# x11-repo (sources.list.d/x11.list)
deb https://packages.termux.org/apt/termux-x11 x11 main
Updatable packages:
All packages up to date
Android version:
9
Kernel build information:
Linux localhost 3.18.120-perf-ge48a7e0 termux/termux-packages#1 SMP PREEMPT Wed Nov 4 13:03:30 CST 2020 armv8l Android
Device manufacturer:
Xiaomi
Device model:
Redmi 6 Pro

```

**Comment by xeffyr** [comment]
Did it myself, strace output is limited to the line of interest:
```
openat(AT_FDCWD, "/data/data/com.termux/files/usr/share/lxappearance/ui/lxappearance.ui", O_RDONLY) = -1 ENOENT (No such file or directory)
```

*Every bug report is a collaboration and not thrown-in list with content similar to "hey, it doesn't work, just shows nothing". I usually either close/delete or just not review empty bug reports as they in most cases just waste of time. It is always possible to obtain execution logs, even if program produces empty output.*

**Comment by xeffyr** [comment]
Package will be fixed in `0.6.3-3` which will be available shortly on package.termux.org and within hours on mirrors.
